yarn.lock

```diff
@@ -10228,6 +10228,13 @@ d3-time-format@2:
   dependencies:
     d3-time "1 - 2"
 
+d3-time-format@^2.1.3:
```

Comment:
can you try running yarn dedupe. Or changing the d3-time-format version to match the package dependencies mentioned in line 10217 or 10224.

Reply:
Ok